nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -158,6 +148,7 @@ public void suggestNullableReturnSimpleTest() {
                 temporaryFolder.getRoot().getAbsolutePath(),
                 "-XepOpt:NullAway:AnnotatedPackages=com.uber",
                 "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:SerializeFixMetadataVersion=2",
```

Comment:
Actually, for tests that aren't explicitly about a particular serialization version, let's remove this and and have it use the latest by default. That will make it easier to detect any effect on the tests for future serialization version changes. It might still be worth having some version specific sanity tests, of course, but for the vast majority of test cases, we should just have it default to the latest.

Reply:
Sure, sounds good. Removed explicitly setting to version `2`. We can add specific unit tests for version `2` once we have version `3` to verify backward compatibility, just like how we did it for `1`.